.circleci/config.yml

```diff
@@ -195,6 +195,7 @@ commands:
                   if [[ -n ${CIRCLE_BRANCH+x} ]] && [[ $CIRCLE_BRANCH != "main" ]] && [[ "$(git log -1)" = *"publish_android_snapshot"* ]]; then
                     sed -i -e "s/^VERSION_NAME=.*/VERSION_NAME=${LAST_RELEASE:9}-${CIRCLE_BRANCH}-SNAPSHOT/" gradle.properties
                     curl -Ls https://mapbox-release-engineering.s3.amazonaws.com/mbx-ci/latest/mbx-ci-linux-amd64 > mbx-ci && chmod 755 ./mbx-ci
+                    head -n 1 gradle.properties
```

Comment:
Why do we need it?

Reply:
If I understand correctly we just print first line from `gradle.properties` - is it used somehow afterwards?

Reply:
If you build a snapshot, you can easy copy the version name in CI and paste in your project

Reply:
I think `make sdkRegistryUpload` task will also print the artifact name of the snapshot.

Reply:
Oh right, haven't seen that before.